scripts/install.sh

```diff
@@ -101,14 +93,8 @@ del_cli_tab_completion() {
 }
 
 copy_manpages() {
-    if [ -d ${SRC_MANPAGE_PATH} ] && [ "$(ls -A ${SRC_MANPAGE_PATH})" ]; then
-    #if [ -d ${SRC_MANPAGE_PATH} ] && [ -z "$(find ${SRC_MANPAGE_PATH} -mindepth 1 -maxdepth 1)" ]; then
-        echo "  Copying \"${SRC_MANPAGE_PATH}/*\" to \"${DST_MANPAGE_PATH}/.\""
-        rm ${DST_MANPAGE_PATH}/bpfman*.1  &>/dev/null
-        cp ${SRC_MANPAGE_PATH}/bpfman*.1 ${DST_MANPAGE_PATH}/.
-    else
-        echo "  CLI Manpage files not generated yet. Use \"cargo xtask build-man-page\" to generate."
-    fi
+    del_manpages
+    ${BIN_BPFMAN} man ${DST_MANPAGE_PATH}
```

Comment:
Below, where `copy_cli_tab_completion` and `copy_manpages` are called, moved those calls to the end of the function. If `setup.sh install` is called after a `setup.sh uninstall`, there is no bpfman CLI binary installed and the commands fail.

Move these down:

```
    echo "Copy CLI TAB Completion files:"
    copy_cli_tab_completion

    echo "Copy Manpage files:"
    copy_manpages
```

Reply:
Done